contracts/gelato_helpers/GelatoMultiCall.sol

```diff
@@ -3,33 +3,36 @@ pragma solidity ^0.6.10;
 pragma experimental ABIEncoderV2;
 
 import {IGelatoCore, TaskReceipt} from "../gelato_core/interfaces/IGelatoCore.sol";
+import {IGelatoSysAdmin} from "../gelato_core/interfaces/IGelatoSysAdmin.sol";
 import {GelatoTaskReceipt} from "../libraries/GelatoTaskReceipt.sol";
+import {GelatoString} from "../libraries/GelatoString.sol";
 
 /// @title GelatoMultiCall - Aggregate results from multiple read-only function calls on GelatoCore
-/// @author Hilmar X (inspired by Maker's Multicall)
+/// @author Hilmar X & gitpusha (inspired by Maker's Multicall)
 contract GelatoMultiCall {
 
     using GelatoTaskReceipt for TaskReceipt;
-
-    IGelatoCore public immutable gelatoCore;
-
-    constructor(IGelatoCore _gelatoCore) public { gelatoCore = _gelatoCore; }
+    using GelatoString for string;
 
     struct Reponse { uint256 taskReceiptId; string response; }
 
+    address public immutable gelatoCore;
```

Comment:
Why do you remove `IGelatoCore` from the immutable and do it within the function? I find it less cleaner because more code is in put in the functions

Reply:
`IGelatoCor` doesnt have `gelatoMaxGas` fn required by `getGasLimit`